html/modules/custom/iasc_content/iasc_content.module

```diff
@@ -1094,3 +1095,17 @@ function iasc_content_fix_entities($entity_type, $bundle, $field_name) {
   unset($map[$field_name]['bundles'][$bundle]);
   $field_map_kv_store->set($entity_type, $map);
 }
+
+/**
+ * Implemtents hook_views_pre_render().
+ */
+function iasc_content_views_pre_render(ViewExecutable $view) {
+  if ($view->id() === 'iasc_group_content') {
+    $group_id = $view->args[0];
+    if ($group_id) {
```

Comment:
`!empty()` is nice, I find. But oh well :-)

Reply:
Duh ;-)